core/storage/question/gae_models.py

```diff
@@ -573,13 +575,11 @@ class QuestionSummaryModel(base_models.BaseModel):
 
     A QuestionSummaryModel instance stores the following information:
 
-    creator_id, question_model_last_updated, question_model_created_on,
+    question_model_last_updated, question_model_created_on,
     question_state_data.
 
     The key of each instance is the question id.
     """
-    # The user ID of the creator of the question.
-    creator_id = ndb.StringProperty(required=True, indexed=True)
```

Comment:
Do we have any models in production with this data? (Maybe we have to clean up.)

Reply:
I've mailed @seanlip with the list of models to be deleted and modified for clean up.